net/src/p2p_network.rs

```diff
@@ -2,22 +2,19 @@
 //! P2pNetwork instances take a json configuration string
 //! and at load-time instantiate the configured "backend"
 
-use holochain_core_types::json::JsonString;
-
 use holochain_net_connection::{
     net_connection::{NetConnection, NetHandler, NetWorker},
     net_connection_thread::NetConnectionThread,
     protocol::Protocol,
     NetResult,
 };
 
-use super::{ipc_net_worker::IpcNetWorker, mock_worker::MockWorker};
-
-use serde_json;
+use super::{ipc_net_worker::IpcNetWorker, mock_worker::MockWorker, p2p_config::*};
 
 /// The p2p network instance
 pub struct P2pNetwork {
-    con: NetConnectionThread,
+    connection: NetConnectionThread,
+    // config: P2pConfig,
```

Comment:
why commented out?

Reply:
Wasn't sure if it would be useful to keep the config around.